Sources/FusionNFC_Android/NFCManager+Android.swift

```diff
@@ -31,9 +31,37 @@ extension NFCManager: NFCManagerProtocol {
 		enableNfcForegroundDispatch()
     }
     
+    func getStandardURL(url: URL, urlType: URLType) -> URL? {
```

Comment:
Is it possible to attach the getStandardURL through an extension of the NFCManagerProtocol in the _Common module ? The implementation seems for me the same, do both implementations use the URL from Foundation ?